src/test/java/com/puppycrawl/tools/checkstyle/utils/CommonUtilTest.java

```diff
@@ -42,6 +53,9 @@
 
     /** After appending to path produces equivalent, but denormalized path. */
     private static final String PATH_DENORMALIZER = "/levelDown/.././";
+    private static final String PACKAGE_DIR_NAME =
+        CommonUtilTest.class.getPackage().getName().replaceAll("\\.", "/");
+    private static final String SIMPLE_CLASS_NAME = CommonUtilTest.class.getSimpleName();
```

Comment:
I would say get rid of these and just hardcode the value for simple class name and inside `getPackageLocation`.

Reply:
Done.